nautobot_golden_config/migrations/0020_convert_dynamicgroup_part_2.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.14 on 2022-07-11 14:18
+
+from django.db import migrations, models
+from django.utils.text import slugify
+import django.db.models.deletion
+
+
+def create_dynamic_groups(apps, schedma_editor):
```

Comment:
Just want to make sure we mimic what is in place currently. Meaning, if empty is everything before, it migrates to everything after. I don't know what this will do, but just making sure we are purposeful about that.

Reply:
Correct, with 1.4.0 filter of `{}` fails open so IF a scope is not set it will create a group with a filter of `{}` which will be equate to `Device.objects.all()`